Conform text of GPL version 1 to latest FSF version

jkeenan opened the pull request:
Compare to https://www.gnu.org/licenses/old-licenses/gpl-1.0.txt

Conform to pod formatting rules

Fixes: GH #22952

---------------------------------------------------------------------------------
* This set of changes does not require a perldelta entry.

jkeenan committed:
Conform text of GPL version 1 to latest FSF version

Compare to https://www.gnu.org/licenses/old-licenses/gpl-1.0.txt

Conform to pod formatting rules

Fixes: GH #22952

jkeenan commented:
As to the merits of this pull request, see https://github.com/Perl/perl5/issues/22952#issuecomment-2838426121.

vaitkus commented:
I did not notice it back when I filed the issue, but there seems to be at least one other copy of the GPL version 1 in this repository included in the `Copying` file. Also the `README` file includes the old FSF address.

Not sure if these changes should be bundled into this PR, however, they do seem to revolve around the same issue.

Grinnz commented:
I think that the license text needs to be indented by one space, as it was before, in order to keep the whole text as a verbatim block in POD.

Leont commented on t/porting/known_pod_issues.dat:
```diff
@@ -661,9 +661,14 @@ perl5380delta
 perl5381delta
 perl5382delta
 perl5383delta
+perl5384delta
```
These additions seem unrelated to the GPL changes.

jkeenan replied:
Please see https://github.com/Perl/perl5/pull/23231.  A step was missed during recent releases, it appears.

jkeenan committed:
Paste in text from FSF website

Indent all text 1 space for verbatim.  Remove 3 non-printing control
characters found in FSF website to guarantee 'pod2text' will work.
But empty all lines that are all whitespace, so as to reduce number of
lines that have to be mentioned in known_pod_issues.dat.

jkeenan commented:
I re-did the pull request perl @Leont and @Grinnz feedback.  I copied and pasted from the FSF website, though I had to eliminate 3 non-printing characters from that site.  However this p.r. will have to wait for merging until https://github.com/Perl/perl5/pull/23231 is merged into blead.  Then we will have to (1) regenerate the known-pod-issues database; (2) squash the commits in this branch before merging.

ap commented:
> It seems that some of the single quotes got reverted back to backticks (' -> `) during the update.
> 
> Not sure if this is still relevant, but these backticks were viewed as an issue at some point (see issue #7945).

Yes.

And un-backticking those quotes would have meant accepting the rewrapping of the text for those hunks, without however reaping the benefit of eliminating the delta against upstream, due to the different quotes. At that point, applying that rewrapping seems silly… which then makes the other rewrapping hunks questionable…

Adding to that, there were several more copies of the old version of the text strewn across the source tree, which this PR didn’t touch. Some of those had various (locally sensible) textual deltas from upstream already, so they couldn’t just be overwritten with the upstream version at all.

All of that drove us to just apply the textual upstream changes, with only a minimal diff, but everywhere (except in CPAN-upstream stuff, of course – feel free @vaitkus to file another ticket against ExtUtils::CBuilder) – which was easier to just do as a quick separate PR.
